pkg/api/graph/interfaces.go

```diff
@@ -107,5 +105,5 @@ func (m ByKey) Less(i, j int) bool {
 type Suggestion string
 
 func (s Suggestion) String() string {
-	return fmt.Sprintf("try: %s", string(s))
```

Comment:
Most of the information provided in this PR is information not directly useful to a user. We may need an additional field (`AdditionalInfo` or some such) in the marker that will hold that? I would vote for keeping `try:` to provide one-liners that users can cp and use.